src/lib/Renderer.js

```diff
@@ -1067,23 +1066,6 @@ var Renderer = Base.extend('Renderer', {
         return this.buttonCells[key] === true;
     },
 
-    startAnimator: function() {
```

Comment:
Apparently not. There was that comment `// animate the dashed line a bit here for fun` that appeared in several places. Apparently steve did this "for fun" at one point but then stopped using it. Looks like it made a moving "marquee."